src/core/evaluator.js

```diff
@@ -2652,6 +2659,35 @@ class PartialEvaluator {
               })
             );
             return;
+          case OPS.beginMarkedContent:
+            if (includeMarkedContent) {
+              textContent.items.push({
+                type: "beginMarkedContent",
+                tag: isName(args[0]) ? args[0].name : null,
+              });
+            }
+            break;
+          case OPS.beginMarkedContentProps:
+            if (includeMarkedContent) {
+              flushTextContentItem();
+              textContent.items.push({
+                type: "beginMarkedContentProps",
+                id:
+                  isDict(args[1]) && args[1].has("MCID")
+                    ? `page${pageObjId}_mcid${args[1].get("MCID")}`
```

Comment:
With the changes suggested above, this would become
```suggestion
                    ? `${this.idFactory.getPageObjId()}_mcid${args[1].get("MCID")}`
```